Dockerfile

```diff
@@ -8,11 +8,12 @@ RUN \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/*
 
-ADD requirements.txt /app/
-RUN pip install --upgrade pip
-RUN pip install -r /app/requirements.txt
-RUN pip install -r /app/requirements_dev.txt
-
+# Install Python dependencies
+# Pipenv is installed with curl to avoid using pip in "user scheme" mode. See:
+# https://pipenv.readthedocs.io/en/latest/install/#crude-installation-of-pipenv
+RUN curl https://raw.githubusercontent.com/kennethreitz/pipenv/master/get-pipenv.py | python
```

Comment:
I'd like to share a few remarks on that using ```pipenv install --system --deploy``` replaces ```pip install -r requirements.txt|requirements_dev.txt```.

1. It looks like currently Pipenv does not support multiple python versions. So using ```pipenv install``` may fail in other python versions which is different than the required one. So it is better not to use ```pipenv install``` in CI/CD tests. But it is fine in Dockerfile, since python version in docker deploy is and should be always fixed.

2. ```pipenv install``` is slower than ```pip install -r requirements```. But it is still fine if deployment time is not in first priority.